src/USD.NET/serialization/UsdIo.cs

```diff
@@ -203,7 +203,7 @@ public class UsdIo {
 
       PropertyInfo[] properties = Reflect.GetCachedProperties(t.GetType());
       FieldInfo[] fields = Reflect.GetCachedFields(t.GetType());
-      object value = t;
+      object value = t;  // TODO: is this variable useful? Doesn't seem to be used. See line 245 bellow as well.
```

Comment:
I don't know what my crazy logic was here, I would try removing this and running the Tests project in the USD.NET solution.